core/src/main/resources/xsl/builder.xsl

```diff
@@ -570,11 +581,10 @@
         <xsl:variable name="header_next" as="xsd:string*">
             <xsl:choose>
                 <xsl:when test="$headers">
-                    <xsl:value-of select="(for $h in $headers[@name = $names[1]] return check:HeaderID($h),
-                                          check:HeaderFailID($headers[@name = $names[1]][1]))"/>
+                    <xsl:sequence select="check:getNextHeaderLinksByName($headers, $headers[1]/@name)"/>
                 </xsl:when>
                 <xsl:otherwise>
-                    <xsl:value-of select="$next"/>
+                    <xsl:sequence select="$next"/>
```

Comment:
You'll notice header_next is a sequence of strings.  When I said value-of it returns a single value "987  889" ...when I use sequence it returns a sequence "987", "889".

In practice it didn't matter before, but it does now.